### PR TITLE
UI modification for creating robot account 

### PR DESCRIPTION
**Pull request by AllForNothing**
#8083 
Signed-off-by: sshijun <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts [hunk 1]
```diff
@@ -185,4 +185,15 @@ export class AddRobotComponent implements OnInit, OnDestroy {
         this.messageHandlerService.showSuccess(res);
       });
   }
+
+  radioChange(num): void {
+    if (num === 1) {
+      this.robot.access.isPullImage = true;
+      this.robot.access.isPushOrPullImage = false;
+    }
+    if ( num === 2) {
```
Comment (jwangyangls) [inline src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: why not use else if

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts [hunk 2]
```diff
@@ -185,4 +185,15 @@ export class AddRobotComponent implements OnInit, OnDestroy {
         this.messageHandlerService.showSuccess(res);
       });
   }
+
+  radioChange(num): void {
```
Comment (jwangyangls) [inline src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: the parameter 'num'? I think it is not a good way to extends function.

Comment (AllForNothing) [reply src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: The parameter 'num' is a simple mark to know which radio is checked

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts (hunk 1 above)
Comment (AllForNothing) [reply src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: one 'if'  to one 'radio button', easy to read.

Comment (jwangyangls) [reply src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: These two IF are mutually exclusive,If num = 1    I don't need to execute if num = 2

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts (hunk 2 above)
Comment (jwangyangls) [reply src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: I think it is not standard to use 1 or 2.If we add new permisson, do we need to add 3, 4, 5?

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts (hunk 1 above)
Comment (zhoumeina) [reply src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: I agree with wangyang's comments, please use if ,else if instead. And please do not use magic number in the code. const definition will be preferred.

Comment (AllForNothing) [reply src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: modified .Thanks for your comments

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts (hunk 2 above)
Comment (AllForNothing) [reply src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: modified. Thanks a lot.

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.scss [hunk 3]
```diff
@@ -35,3 +35,22 @@
 .no-margin {
   margin: 0;
 }
+.display-table{
+  display: table;
```
Comment (zhoumeina) [inline src/portal/src/app/project/robot-account/add-robot/add-robot.component.scss]: please use bootstrap grid system to define the layout.

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts [hunk 4]
```diff
@@ -185,4 +185,14 @@ export class AddRobotComponent implements OnInit, OnDestroy {
         this.messageHandlerService.showSuccess(res);
       });
   }
+
+  radioChange(radioId: string ): void {
```
Comment (zhoumeina) [inline src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: please add this logic to onSubmit() method

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.html [hunk 5]
```diff
@@ -31,58 +31,57 @@ <h3 class="modal-title">{{'ROBOT_ACCOUNT.CREAT_ROBOT_ACCOUNT' | translate}}</h3>
...
+                  </label>
+              </div>
+              <div class="clr-col">
+                  <div class="form-group padding-left-120">
+                      <label>{{'ROBOT_ACCOUNT.PERMISSIONS_IMAGE' | translate}}</label>
+                      <div class="radio-inline">
+                          <input type="radio" name="image-permission"
+                                 id="image-permission-push"
+                                 value="push"
+                                 [(ngModel)]="imagePermission">
```
Comment (zhoumeina) [inline src/portal/src/app/project/robot-account/add-robot/add-robot.component.html]: Please modify "push" to "PUll"

**Pull request closed without merging**